core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -115,14 +115,16 @@ type EthBroadcaster[ADDR types.Hashable[ADDR], TX_HASH types.Hashable[TX_HASH],
 	// triggers allow other goroutines to force EthBroadcaster to rescan the
 	// database early (before the next poll interval)
 	// Each key has its own trigger
-	triggers map[string]chan struct{}
+	triggers map[ADDR]chan struct{}
 
 	chStop chan struct{}
 	wg     sync.WaitGroup
 
 	initSync  sync.Mutex
 	isStarted bool
 	utils.StartStopOnce
+
+	addrParser func(string) ADDR
```

Comment:
nit/ `addrParser` sounds like the name of an interface with a `ParseAddress()` method. What about `parseAddr` or `addrParseFn`?
```suggestion
	parseAddr func(string) ADDR
```
```suggestion
	addrParseFn func(string) ADDR
```

Reply:
sure! i'll use `parseAddr`